src/webgl.js

```diff
@@ -1593,12 +1590,12 @@ export function main(player, canvas, options) {
 
     drawScene(gl, render_program_info, buffers);
 
-    var now = new Date().getTime();
+    now = new Date().getTime();
```

Comment:
Should this be `const now = new Date().getTime();` ?

Reply:
Good question. I didn't mean to include that in this PR, the `now` var is passed to the render function, but it's never used until this section where we set the now value before use anyway. So we can't use `const new` because it's redeclaring `now`, even though the first `now` is useless.